### PR TITLE
Fix: Update status for `Same-site cookies` to `Shipped`

### LINKED ISSUE #616: Change status of SameSite cookie
Edge shipped SameSite cookie with 6B update. Let's change the status to shipped!

### PR DESCRIPTION
**Pull request by shhnjk**
Close #616

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -58,11 +58,16 @@
...
-    "link": "https://tools.ietf.org/html/draft-west-first-party-cookies-07",
+    "link": "https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00",
     "summary": "Defines a SameSite attribute, allowing a server to specify that cookies should not be sent with cross-site requests.",
     "standardStatus": "Editor's draft",
     "ieStatus": {
-      "text": "In Development"
+      "text": "Shipped",
+      "ieUnprefixed": "11"
+    },
+    "ff_views": {
```
Comment (alrra) [inline status.json]: This part can be removed as it come from [Chrome status](https://www.chromestatus.com/features/4672634709082112).

Comment (shhnjk) [reply status.json]: What about IE status? We also shipped SameSite cookies in IE 11.

File: status.json [hunk 2]
```diff
@@ -58,11 +58,12 @@
...
     "name": "Same-site cookies",
     "category": "Security",
-    "link": "https://tools.ietf.org/html/draft-west-first-party-cookies-07",
+    "link": "https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00",
     "summary": "Defines a SameSite attribute, allowing a server to specify that cookies should not be sent with cross-site requests.",
     "standardStatus": "Editor's draft",
     "ieStatus": {
-      "text": "In Development"
+      "text": "Shipped",
+      "ieUnprefixed": "11"
```
Comment (alrra) [inline status.json:66]: @kypflug Can you confirm this? Thanks!

Comment (kypflug) [reply status.json:66]: Confirmed! 👍

**Comment by alrra** [comment]
Thanks, @shhnjk!

**Comment by mkurz** [comment]
Can someone please clarify which Edge versions exactly will have "Same-site cookie" support? When will this be shipped to which versions? When will the update roll out to older Windows 10 builds?
According to https://developer.microsoft.com/en-us/microsoft-edge/platform/status/samesitecookies it 
 is shipped to 10240+ which means **all** Edge versions since it's birth will have/get support? Sure?
See https://en.wikipedia.org/wiki/Windows_10#Updates_and_support

**Comment by alrra** [comment]
> Can someone please clarify which Edge versions exactly will have "Same-site cookie" support?

For people stumbling over this PR, the discussion around the above is in: https://github.com/MicrosoftEdge/Status/issues/616.

**Comment by molant** [comment]
@mkurz see the note in the comments:

image

It's only Edge 16. Another option would be to do the opposite: Update the note to say that on top of Edge 16, IE11 is also supported.

Unfortunately we cannot specify IE11 and a different version of Edge at the moment.

**Pull request closed without merging**